src/lib/browserLocale.ts

```diff
@@ -1,4 +1,8 @@
+import { DEFAULT_FIAT_CURRENCY } from 'constants/Config'
+import { getCurrency } from 'locale-currency'
```

Comment:
Do we want a package for this? Could simply have the map in a constants file?

Reply:
I felt like we didn't want to have to do this maintenance as, despite what we might think, it can change from time to time: https://github.com/tadeegan/locale-currency/commits/master

But I'm fine moving the map and the getCurrency function otherwise.

Reply:
Yeah the repo has been updated but the package itself hasn't been published in 6 years 😞 
<img width="165" alt="image" src="https://github.com/shapeshift/web/assets/17035424/528f9fbd-644f-466c-8224-dbaa87811244">

Reply:
would probably get the latest mapping from the repo, move it to constants and call it a day, since the package is inactive and even if it was maintained, it isn't really going to be a concern for us

Reply:
You're right. Removed dependency and added the map and needed functions in it (as they are only used to access the constants).